docs/attributes-registry/process.md

```diff
@@ -15,6 +15,7 @@ An operating system process.
 
 | Attribute                     | Type     | Description                                                                                                                                                                                                                                                                                                                                                 | Examples                                            | Stability                                                        |
 | ----------------------------- | -------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------- | ---------------------------------------------------------------- |
+| `process.args_count`          | int      | Length of the process.command_args array [1]                                                                                                                                                                                                                                                                                                                | `4`                                                 | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
This is also more like a metric, no? 🤔

Reply:
This could be a metric, but to me, it doesn't feel like it should. I don't think it represents any measurement that anyone would care to track over time with a metric. The arg_count generally won't change (outside of an exec), and it's not a measurement that indicates system performance or utilization.

It does have value as an attribute in a log/event though. This is intended to be a convenience/performance field. You could also get arg count by counting the `process.args` length, but having a separate `arg_count` field makes it easier to write queries on this field directly.

Reply:
>The arg_count generally won't change (outside of an exec), and it's not a measurement that indicates system performance or utilization.

Hum, but this then contradicts the brief of the field, no?

> This field can be useful for querying or performing bucket analysis on how many
> arguments were provided to start a process. More arguments may be an indication
> of suspicious activity.

If it's important to recognize suspicious activity like that, I feel having it in a log or span, rather than being a metric is more cumbersome? Because one would have to extract the attribute from logs/spans and then plot it somehow as a metric?

Reply:
I've talked with the team at Elastic that uses this field to work on detections, and they've told me it has value as both a metric and an attribute attached to an event.

I've now added it as both a metric and a field in the attribute registry in this PR. Although I didn't see any prior art for a field being used as both, so I'm not sure if I've done it in the correct way

Reply:
Hum I'm also not sure, having both the attribute and the metric seems a bit.. unnecessary but I don't have strong opinions against it. I left another comment because you used a Histogram for it and I'm not sure why.